src/Accounts.js

```diff
@@ -0,0 +1,517 @@
+/* eslint-disable max-len */
+
+import { each, keys, includes, trim, isEmpty, merge, extend } from 'lodash';
+import { Iterable } from 'immutable';
+import createStore from './createStore';
+
+const ACCESS_TOKEN = 'js-accounts:accessToken';
+const REFRESH_TOKEN = 'js-accounts:refreshToken';
+
+const PATH = 'js-accounts/';
+const LOGIN = `${PATH}LOGIN`;
+const CLEAR_FORM = `${PATH}CLEAR_FORM`;
+const ADD_ERROR = `${PATH}ADD_ERROR`;
+const SET_LOADING = `${PATH}SET_LOADING`;
+const SET_FIELD = `${PATH}SET_FIELD`;
+const CLEAR_ERRORS = `${PATH}CLEAR_ERRORS`;
+const SET_USER = `${PATH}SET_USER`;
+
+const initialState = {
+  formType: 'login',
+  isLoading: false,
+  forms: {
+    login: {
+      fields: {
+        user: {
+          value: '',
+          errors: [],
+        },
+        password: {
+          value: '',
+          errors: [],
+        },
+      },
+      errors: [],
+    },
+    signupForm: {
+
+    },
+  },
+  user: null,
+};
+
+const reducer = (state = initialState, action) => {
+  let nextState = state;
+  switch (action.type) {
+    case LOGIN: {
+      break;
+    }
+    case CLEAR_FORM: {
+      const { form } = action.payload;
+      nextState = extend({}, state, {
+        forms: {
+          [form]: initialState.forms[form],
+        },
+      });
+      break;
```

Comment:
Style nitpick - I prefer to avoid `let` and `break` and return directly from each case, but that may be a style choice on your part in this package.